checkpoint_quantum_firewall/assets/logs/checkpoint-quantum-firewall.yaml

```diff
@@ -0,0 +1,946 @@
+          enabled: true
+          sources:
+            - event_type
+          sourceType: attribute
+          target: checkpoint.quantum.firewall.event_type
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `host_type` to `checkpoint.quantum.firewall.host_type`
+          enabled: true
+          sources:
+            - host_type
+          sourceType: attribute
+          target: checkpoint.quantum.firewall.host_type
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `os_name` to `checkpoint.quantum.firewall.os_name`
+          enabled: true
+          sources:
+            - os_name
+          sourceType: attribute
+          target: checkpoint.quantum.firewall.os_name
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `dst` to `checkpoint.quantum.firewall.dst`
+          enabled: true
+          sources:
+            - dst
+          sourceType: attribute
+          target: checkpoint.quantum.firewall.dst
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `src` to `checkpoint.quantum.firewall.src`
+          enabled: true
+          sources:
+            - src
+          sourceType: attribute
+          target: checkpoint.quantum.firewall.src
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: Remapper for Username Attribute
+      enabled: true
+      filter:
+        query: "@product:DLP"
+      processors:
+        - type: attribute-remapper
+          name: Map `user` to `usr.name`
+          enabled: true
+          sources:
+            - user
+          sourceType: attribute
+          target: usr.name
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `reject_category` to `checkpoint.quantum.firewall.reject_category`
+          enabled: true
+          sources:
+            - reject_category
+          sourceType: attribute
+          target: checkpoint.quantum.firewall.reject_category
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: String Building for Source and Destination IP Port
+      enabled: true
+      filter:
+        query: '@product:("Firewall" OR "IPS" OR "Anti-Spam and Email Security" OR "URL
+          Filtering") '
+      processors:
+        - type: string-builder-processor
+          name: "Define `checkpoint.quantum.firewall.src_ip_port` equal to
```

Comment:
For my reference, would you mind explaining what is the value of having the combination of the source and the port together ?

Reply:
The value would be looking like 10.0.0.1:4545